### PR TITLE
regen for autorest.csharp#2419

### PR DESCRIPTION
**Pull request by ArcturusZhang**
Regeneration for https://github.com/Azure/autorest.csharp/pull/2419

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/network/Azure.ResourceManager.Network/src/autorest.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ rename-mapping:
   TrafficAnalyticsConfigurationProperties.trafficAnalyticsInterval: TrafficAnalyticsIntervalInMinutes
   TroubleshootingParameters.properties.storagePath: storageUri
   ProtocolConfiguration.HTTPConfiguration: HttpProtocolConfiguration
-  FlowLogFormatParameters: FlowLogProperties
+  FlowLogFormatParameters: FlowLogFormat
```
Comment (ArthurMa1978) [inline sdk/network/Azure.ResourceManager.Network/src/autorest.md:38]: you reverted my change of https://github.com/Azure/azure-sdk-for-net/pull/29668

Comment (ArcturusZhang) [reply sdk/network/Azure.ResourceManager.Network/src/autorest.md:38]: I will fix this

**Review by ArthurMa1978 — APPROVED** [review]

**Comment by ArcturusZhang** [comment]
Tired of constantly reruning these pipelines. They will eventually pass, but they usually have various of glitches.

**Pull request merged**